src/components/standalone/dns_dhcp/LeasesTable.vue

```diff
@@ -25,6 +26,11 @@ const props = defineProps<{
   showDynamicLeases: boolean
 }>()
 
+const displayLeases = computed(() => {
+  // render reactive leases
+  return props.leases
+})
```

Comment:
Props are reactive by definition, what happened here?

Reply:
the leasesTable did not render the changes from {dynamic, static}Lease until I made it reactive

Reply:
yes I did some attempts and debug with AI, props.leases is well sent from parent to child, but useSort expects a computed value to follow the changes like I proposed